submodules/LIB/hardware/modules/iob_interface/README.md

```diff
@@ -1,3 +1,30 @@
-# IOb-interface #
+# IOb-interface
 
-![Read Transaction](iob_if_read.png "Read Transaction")
+The IOb-interface has the following signals:
+- `clk_i`: interface clock
+- `valid_i`: source signal valid transaction
+- `addr_i`: address for access
+- `wdata_i`: data to write
+- `wstrb_i`: write strobe
+- `rdata_o`: read data
+- `rvalid_o`: valid read data
+- `ready_o`: destination ready to accept transactions
+
+Transactions occur when both `valid_i` and `ready_o` are HIGH for one `clk_i`
+period.
+
+Use `wstrb_i` to distinguish between READ and WRITE operations. READ operations
+occur when all `wstrb_i` bits are LOW.
+
+## Read Transaction
+For READ operations: when `valid_i` and `ready_o` are HIGH, read `addr_i`
+address. The `rdata_o` is available when `rvalid_o` is HIGH.
```

Comment:
For READ operations: when `valid_i` and `ready_o` are HIGH, address `addr_i` is read. The data on `rdata_o` is available when `rvalid_o` is HIGH.